src/OrchardCore.Modules/OrchardCore.Flows/Drivers/FlowMetadataDisplay.cs

```diff
@@ -34,7 +37,15 @@ public override async Task<IDisplayResult> UpdateAsync(ContentItem contentItem,
                 return null;
             }
 
-            await contentItem.AlterAsync<FlowMetadata>(model => updater.TryUpdateModelAsync(model, Prefix));
+            var viewModel = new FlowMetadataEditViewModel();
+            if (await updater.TryUpdateModelAsync(viewModel, Prefix))
+            {
+                contentItem.Alter<FlowMetadata>(model => {
+                    model.Alignment = viewModel.Alignment;
```

Comment:
You shouldn't need to use `.Alter` here, just `model.Alignment = viewModel.Alignment`